src/components/TheHeader.vue

```diff
@@ -23,6 +23,7 @@
             :data-cy="cyName"
             :title="text"
             :to="page"
+            :style="{margin :'4px'}"
```

Comment:
The intent of the issue is to add a margin between the icon and text, not between the tabs themselves. That means we should move this onto the icon itself and specifically add margin-right. 

Additionally, we normally favor using classes that utilize preexisting Vuetify styles ((docs:)[https://vuetifyjs.com/en/styles/spacing/]), in this case for a right margin of 4px (a single unit of Vuetify's spacing helpers), that would be 
```class="mr-1"```